### PR TITLE
chore(secmaster): add secmaster version action attributes

### PR DESCRIPTION
**Pull request by luoping-12345**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```
add secmaster version action attributes
```

## PR Checklist

* [x] Tests added/passed.

```
make testacc TEST="./huaweicloud/services/acceptance/secmaster" TESTARGS="-run TestAccPlaybookVersioAction_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/secmaster -v -run TestAccPlaybookVersioAction_basic -timeout 360m -parallel 4
=== RUN   TestAccPlaybookVersioAction_basic
=== PAUSE TestAccPlaybookVersioAction_basic
=== CONT  TestAccPlaybookVersioAction_basic
--- PASS: TestAccPlaybookVersioAction_basic (84.74s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/secmaster 84.790s
```

* [x] Documentation updated.
* [x] Schema updated.
* [ ] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

    
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

### CODE REVIEW

File: huaweicloud/services/secmaster/resource_huaweicloud_secmaster_playbook_version_action.go [hunk 1]
```diff
@@ -52,11 +54,130 @@ func ResourcePlaybookVersionAction() *schema.Resource {
 				ConflictsWith: []string{"status"},
 				Description:   `Specifies whether the playbook version is enabled.`,
 			},
+			"version": {
```
Comment (houpeng80) [inline huaweicloud/services/secmaster/resource_huaweicloud_secmaster_playbook_version_action.go:59]: the version_id is determined, this attribute should be removed

File: docs/resources/secmaster_playbook_version_action.md [hunk 2]
```diff
@@ -56,7 +56,64 @@ The following arguments are supported:
...
+* `status` - (Optional, String, NonUpdatable) Specifies the playbook version status. The value can only be **APPROVING**.
 
 * `enabled` - (Optional, Bool, NonUpdatable) Specifies whether the playbook version is enabled.
   The value can be **true**(enable version) or **false**(disable version).
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The data source ID.
```
Comment (houpeng80) [inline docs/resources/secmaster_playbook_version_action.md]:
```suggestion
* `id` - The resource ID.

```

File: docs/resources/secmaster_playbook_version_action.md [hunk 3]
```diff
@@ -56,7 +56,64 @@ The following arguments are supported:
...
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID.
+
+* `version` - The playbook version.
+
+  The [version](#version_struct) structure is documented below.
+
+<a name="version_struct"></a>
+The `versions` block supports:
```
Comment (saf3dfsa) [inline docs/resources/secmaster_playbook_version_action.md]:
```suggestion
The `version` block supports:

```

File: docs/resources/secmaster_playbook_version_action.md [hunk 4]
```diff
@@ -56,7 +56,64 @@ The following arguments are supported:
...
+
+* `id` - The playbook version ID.
```
Comment (saf3dfsa) [inline docs/resources/secmaster_playbook_version_action.md]: id is removed？

**Pull request merged**